An error occurred during AUTOPAUSE & COMMUNITY since PR #835

## Describe the bug

```bash
docker compose --project-directory examples/autopause up
```
<details><summary>Error Log</summary>

```
palworld-server  | [2026-06-12 09:36:48] [INFO] [AUTO PAUSE] Service ... start
palworld-server  | [2026-06-12 09:36:48] [LOG] REST accessed endpoint /v1/api/players OK (x4)
palworld-server  | [2026-06-12 09:37:04] [LOG] REST accessed endpoint /v1/api/save OK
palworld-server  | [2026-06-12 09:37:04] [INFO] [AUTO PAUSE] Paused. (PID:350)
palworld-server  | [2026-06-12 09:37:05] [INFO] curl: unknown --write-out variable: 'exitcode'
palworld-server  | [2026-06-12 09:37:05] [ERROR] [AUTO PAUSE] server/update API call failed: 
palworld-server  | [2026-06-12 09:37:36] [INFO] curl: unknown --write-out variable: 'errormsg'
palworld-server  | [2026-06-12 09:37:36] [INFO] curl: unknown --write-out variable: 'exitcode'
palworld-server  | [2026-06-12 09:37:36] [ERROR] [AUTO PAUSE] server/register API call failed: 
```
</details>

```bash
$ docker exec -it palworld-server curl --version
curl 7.74.0 (x86_64-pc-linux-gnu) libcurl/7.74.0 OpenSSL/1.1.1w zlib/1.2.11 brotli/1.0.9 libidn2/2.3.0 libpsl/0.21.0 (+libidn2/2.3.0) libssh2/1.9.0 nghttp2/1.43.0 librtmp/2.3
Release-Date: 2020-12-09, security patched: 7.74.0-1.3+deb11u16
Protocols: dict file ftp ftps gopher http https imap imaps ldap ldaps mqtt pop3 pop3s rtmp rtsp scp sftp smb smbs smtp smtps telnet tftp 
Features: alt-svc AsynchDNS brotli GSS-API HTTP2 HTTPS-proxy IDN IPv6 Kerberos Largefile libz NTLM NTLM_WB PSL SPNEGO SSL TLS-SRP UnixSockets
```

**cm2network/steamcmd:root**

|Trend|Base Image|curl version|AUTOPAUSE|
|--|--|--|--|
|NOW|Trixie (Debian 13)|8.14.1|:o:|
|PREV|Bookworm (Debian 12)|7.88.1|:o:|
|OLD|Bullseye (Debian 11)|7.74.0|:x:|

To obtain more information about api.palworldgame.com, we will need to use curl version 7.75 or later.
And `--json` option requires 7.82 or later.

Why downgrade to Bullseye (Debian 11)?


## To Reproduce

```bash
docker compose --project-directory examples/autopause up
```

## Expected behavior

```
palworld-server  | [2026-06-12 10:13:12] [INFO] [AUTO PAUSE] Service ... start
palworld-server  | [2026-06-12 10:13:12] [LOG] REST accessed endpoint /v1/api/players OK (x4)
palworld-server  | [2026-06-12 10:13:29] [LOG] REST accessed endpoint /v1/api/save OK
palworld-server  | [2026-06-12 10:13:29] [INFO] [AUTO PAUSE] Paused. (PID:398)
```

## Actual behavior

See section of Describe the bug.

## Screenshots

<!-- If applicable, add screenshots to help explain your problem. -->

## OS information (IMPORTANT)

- OS:  Windows 11 wsl2 Ubuntu 24.04.4 LTS
- Palworld Version:  <!--e.g. 0.1.5.1 -->
- Docker Image Version:  2.4.0

## Hardware information (IMPORTANT)

- CPU: INTEL Core Ultra 9 285
- Memory:  64GB (wsl2: 32GB)

## compose.yaml contents (IMPORTANT)

[examples/autopause/compose.yaml](https://github.com/thijsvanloef/palworld-server-docker/blob/main/examples/autopause/compose.yaml)

## Container/Host Logs (IMPORTANT)

See section of Describe the bug.

## Additional context

Nothing.

## Bug Report Checklist

- [x] I searched for my issue on the [Known issues page](https://palworld-server-docker.loef.dev/known-issues)
- [x] I searched for my issue in the [Closed issues](https://github.com/thijsvanloef/palworld-server-docker/issues?q=is%3Aissue+is%3Aclosed)
- [x] I have included all relevant information required above.
- [x] I am willing to fix this bug

--- a/Dockerfile
+++ b/Dockerfile
@@ -17,7 +17,7 @@ RUN wget -q https://github.com/gorcon/rcon-cli/archive/refs/tags/v${RCON_VERSION
     && rm -rf rcon-cli-${RCON_VERSION} \
     && go build -v ./cmd/gorcon
 
-FROM cm2network/steamcmd:root-bullseye AS base-amd64
+FROM cm2network/steamcmd:root AS base-amd64
 # Ignoring --platform=arm64 as this is required for the multi-arch build to continue to work on amd64 hosts
 # hadolint ignore=DL3029
 FROM --platform=arm64 sonroyaalmerol/steamcmd-arm64:root-bookworm-2026-06-07 AS base-arm64

--- a/scripts/autopause/community/functions.sh
+++ b/scripts/autopause/community/functions.sh
@@ -26,14 +26,13 @@ APComm_formatLogMessage() {
 
 # api.palworldgame.com/server Call
 APComm_API() {
-    local api="${1}"
-    local data="${2}"
-    local url="${APComm_API_URL}${api}"
-    local accept="Accept: application/json"
-    local agent="X-UnrealEngine-Agent"
+    local -r api="${1}"
+    local -r data="${2}"
+    local -r url="${APComm_API_URL}${api}"
+    local -r agent="X-UnrealEngine-Agent"
     local raw response meta exit_code http_code error_message
 
-    raw=$(curl -s --fail --max-time 10 --retry 2 --retry-delay 3 -L -X POST "${url}" -H "${accept}" -H "Content-Type: application/json" -A "${agent}" -d "${data}" --write-out "\n${APComm_curl_meta_sep}\n%{exitcode}\n%{response_code}\n%{errormsg}")
+    raw=$(curl -s --fail --max-time 10 --retry 2 --retry-delay 3 -L -X POST "${url}" -A "${agent}" --json "${data}" --write-out "\n${APComm_curl_meta_sep}\n%{exitcode}\n%{response_code}\n%{errormsg}")
     meta="${raw##*$'\n'"${APComm_curl_meta_sep}"$'\n'}"
     response="${raw%$'\n'"${APComm_curl_meta_sep}"$'\n'*}"
 

--- a/scripts/helper_functions.sh
+++ b/scripts/helper_functions.sh
@@ -215,16 +215,15 @@ REST_API() {
     local -r api="${1}"
     local -r data="${2}"
     local -r url="http://localhost:${REST_API_PORT}/v1/api/${api}"
-    local -r accept="Accept: application/json"
     local -r userpass="admin:${ADMIN_PASSWORD}"
     local -r post_api="save|stop"
     local -r down_api="shutdown|stop"
     local -i result=0
     if [ "${data}" = "" ] && [[ ! ${api} =~ ${post_api} ]]; then
-        curl -s -L -X GET  "${url}" -H "${accept}" -u "${userpass}"
+        curl -s -L -X GET  "${url}" -u "${userpass}" -H "Accept: application/json"
         result=$?
     else
-        curl -s -L -X POST "${url}" -H "${accept}" -H "Content-Type: application/json" -u "${userpass}" -d "${data}"
+        curl -s -L -X POST "${url}" -u "${userpass}" --json "${data}"
         result=$?
     fi
     if [ ${result} -eq 0 ] && [[ ${api} =~ ${down_api} ]]; then

